.github/workflows/unix.yml

```
@@ -20,14 +20,44 @@ jobs:
       run: |
         eval $(spack env activate --sh .github/ci/spack-envs/clangtidy_nopy_ompi_h5_ad1_ad2/)
         spack install
```

Just reverting the white space change:
```suggestion
        spack install

```